src/convert/formula.jl

```diff
@@ -38,6 +38,11 @@ end
 
 function rcopy(::Type{FormulaTerm}, l::Ptr{LangSxp})
     expr = rcopy(Expr, l)
+    # special case of a simple variable, like in aes(x, y)
+    if Meta.isexpr(expr, :call) && length(expr.args) == 2 && expr.args[1] == :~
+        return expr
```

Comment:
```suggestion
        return expr.args[2]
```

(so that we drop the tilde)